src/theme/ColorModeToggle/index.js

```diff
@@ -6,12 +6,12 @@ import IconLightMode from "@theme/Icon/LightMode";
 import IconDarkMode from "@theme/Icon/DarkMode";
 import styles from "./styles.module.css";
 import { useColorScheme } from "@mui/material";
-import useDocumentationPath from "@site/src/hooks/useDocumentationPath";
+import usePagePath from "@site/src/hooks/usePagePath";
 function ColorModeToggle({ className, buttonClassName, value, onChange }) {
   const isBrowser = useIsBrowser();
   /* Color scheme switcher from MUI framework. */
   const { setMode } = useColorScheme();
-  const { isDocumentation } = useDocumentationPath();
+  const { isDocumentation, isLanding } = usePagePath();
```

Comment:
Remove redundant variables if not used
```suggestion
  const { isLanding } = usePagePath();
```